kiali/README.md

```diff
@@ -2,12 +2,13 @@
 
 ## Overview
 
-The following instructions outline how to install [`Kiali`](https://github.com/kiali/helm-charts/tree/master/kiali-operator) in Kyma.
+The following instructions outline how to install [`Kiali`](https://github.com/kiali/helm-charts/tree/master/kiali-operator) in Kyma. As Kiali requires a prometheus installation preserving istio metrics, this example assumes that you installed the [custom prometheus example](./../prometheus/).
```

Comment:
```suggestion
The following instructions outline how to install [`Kiali`](https://github.com/kiali/helm-charts/tree/master/kiali-operator) in Kyma. Because Kiali needs a Prometheus installation preserving Istio metrics, this example assumes that you installed the [custom Prometheus example](./../prometheus/).
```
You can drop the 2nd sentence here, it's the 2nd bullet of the Prerequisites, right?

Reply:
We could, yes. Thought it is nicer to have it as a sentence as well to give small "background"